app/models/converters/integer_converter.rb

```diff
@@ -4,6 +4,8 @@
 class IntegerConverter < BaseConverter
   def self.convert(value)
     value = super(value)
+
+    return value unless value.respond_to? :to_i
     value.blank? ? nil : value.to_i
```

Comment:
Curious: what was the Class of a value that we want to return that doesn't have a `#to_i`?